src/sb_utils/include/RvizUtils.h

```diff
@@ -0,0 +1,87 @@
+/*
+ * Created By: Robyn Castro
+ * Created On: November 10th, 2017
+ * Description: Utilities to create Rviz visualisation messages
+ *
+ */
+#ifndef SB_UTILS_RVIZUTILS_H
+#define SB_UTILS_RVIZUTILS_H
+
+// Messages
+#include <geometry_msgs/Point.h>
+#include <visualization_msgs/Marker.h>
+
+class RvizUtils {
```

Comment:
OK, so after some discussion in the slack #software-leads channel, we've decided to place `RvizUtils` under the `snowbots` namespace. After that, this should be good to go.